lms/templates/widgets/cookie-consent.html

```diff
@@ -0,0 +1,28 @@
+<!-- Cookie Consent -->
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _ %>
+<%namespace name='static' file='../static_content.html'/>
+ 
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.js"></script>
+
+<script>
+  var message = "${_('This website uses cookies to ensure you get the best experience on our website.')}";
+  var learn = "${_('Learn more')}";
+  var gotit = "${_('Got it!')}"
+
+  window.cookieconsent.initialise({
+    window: '<div dir="${static.dir_rtl()}" role="dialog" aria-live="polite" aria-label="cookieconsent" aria-describedby="cookieconsent:desc" class="cc-window {{classes}}"><!--googleoff: all-->{{children}}<!--googleon: all--></div>',
```

Comment:
@cahrens, I removed the aria-live attr but I have kept the aria-label in accordance with [these EdX guidelines](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/accessibility.html#manage-the-focus-for-pop-ups). I also made the cookie consent banner keyboard accessible, the div can be escaped and the button is activated with space and enter.